accessing
exception
	"Answer my value (exception) because #content raises 
	an error"
	
	^ value